Packs/Okta/Integrations/Okta_v2/Okta_v2.py

```diff
@@ -496,6 +496,33 @@ def clear_user_sessions(self, user_id):
             resp_type='text'
         )
 
+    def get_zone(self, args, zoneID):
+        uri = f'zones/{zoneID}'
+        return self._http_request(
+            method='GET',
+            url_suffix=uri,
+            resp_type='text'
+        )
+
+    def list_zones(self, args):
+        uri = f'zones'
+        return self._http_request(
+            method='GET',
+            url_suffix=uri,
+            resp_type='text'
```

Comment:
are you sure the response is text and not JSON?
If the response is JSON, then you can delete this line. And it will automatically will convert the response to json